wandb/framework/tensorflow/estimator_hook.py

```diff
@@ -43,6 +42,6 @@ def after_run(self, run_context, run_values):
         print(run_values)
```

Comment:
We actually have a print statement here?  Let's remove.

Reply:
removed